src/api/uploadObjectsToBucket.ts

```diff
@@ -21,7 +35,12 @@ export async function uploadObjectsToBucket(
           Key: keyOfObjectToUpload,
           Body: fs.createReadStream(statsOfObjectToUpload.path)
         })
-        .promise();
+        .promise()) as SendDataWithVersion;
+      emitter.emit('object:upload', keyOfObjectToUpload, {
+        progress: 100,
+        version: VersionId,
+        status: ObjectDiffStatus.ADDED
```

Comment:
status could be `Modified` or `Added` - maybe just remove `ObjectEvent .status` for now to get this merged

Reply:
true! and also i was thinking with progess: 0 does `Modified` or `Added` make sense? I'll go ahead and take it out for now

Reply:
the tense `ed` definitely doesn't make sense at `0` but whether its being added or modified still makes some sense.

`version` is a little weird at `0` too... oh well, that's what breaking changes are for.